domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -884,63 +884,66 @@ public void saveBatch(String table, JsonArray entities, Handler<AsyncResult<Resu
     saveBatch(table, entities, DEFAULT_JSONB_FIELD_NAME, replyHandler);
   }
 
-  public void saveBatch(String table, JsonArray entities, String column, Handler<AsyncResult<ResultSet>> replyHandler) {
+  public void saveBatch(String table, JsonArray entities, String column,
+    Handler<AsyncResult<ResultSet>> replyHandler) {
+
+    client.getConnection(res -> {
+      if (res.failed()) {
+        replyHandler.handle(Future.failedFuture(res.cause()));
+        return;
+      }
+      saveBatch(res, table, entities, column, x -> {
+        res.result().close();
+        replyHandler.handle(x);
+      });
+    });
+  }
+
+  private void saveBatch(AsyncResult<SQLConnection> sqlConnection, String table,
```

Comment:
This should be public.
The corresponding saveBatch method without sqlConnection is already public.